.github/workflows/create-release.yml

```
@@ -0,0 +1,53 @@
+    tags:
+      - 'v*' # Trigger on tags with prefix 'v'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4.1.1
+
+    - name: Generate Release Notes
+      id: release_notes
+      uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)